examples/tcs34725_simpletest.py

```diff
@@ -21,8 +21,9 @@
     try:
         temp = sensor.temperature
         lux = sensor.lux
+        print('Temperature: {0}K Lux: {1}'.format(temp, lux))
     except ZeroDivisionError:
         print("No light to measure")
-    print('Temperature: {0}K Lux: {1}'.format(temp, lux))
+    
```

Comment:
pylint complaining about trailing whitespace on line 27 (which is otherwise a blank line).